docs/“How-to” guides/User Guides/Setup Enigma Locally/Local Setup without Docker.md

```diff
@@ -7,15 +7,15 @@ You should have python3 installed on your system and checked out the enigma repo
 ## Steps
 1. Ensure you have a valid `config.json` present locally.
 
-The default [config.json.sample](https://github.com/browserstack/enigma-public-central/blob/main/config.json.sample) should be sufficient to start.
+The default [config.json.sample](https://github.com/browserstack/enigma/blob/main/config.json.sample) should be sufficient to start.
 
 You can then add module-specific configuration for the modules you want integrated with Enigma.
 For detailed instructions on configuration, follow [this doc](/docs/Configuration%20Guide.md)
 
 2. Add folder `db` in root folder of the repository.
 3. Add a package `access_modules` inside `Access`.
 
-Copy this [file](../../Access/base_email_access/access_modules_init.py) as `__init__.py` of `access_modules`
+Copy this [file](../../../../Access/base_email_access/access_modules_init.py) as `__init__.py` of `access_modules`
```

Comment:
We can add absolute paths instead

```
Copy this [file](/Access/base_email_access/access_modules_init.py) as `__init__.py` of `access_modules`
```
in other places as well

Reply:
Done.